package.json

```diff
@@ -63,7 +63,9 @@
     "mocha": "echo 'use `node scripts/mocha`' && false",
     "sterilize": "grunt sterilize",
     "uiFramework:start": "grunt uiFramework:start",
-    "uiFramework:build": "grunt uiFramework:build"
+    "uiFramework:build": "grunt uiFramework:build",
+    "uiFramework:create": "./node_modules/_bin/yo ./ui_framework/generator-kui/app/component.js",
+    "uiFramework:document": "./node_modules/_bin/yo ./ui_framework/generator-kui/app/documentation.js"
```

Comment:
You for sure do not need the whole `node_modules` part, you can just use `yo`. The beauty of npm scripts is that all the modules are injected into the $PATH.

I can't speak to the path following the command though.